### DISCUSSION TITLE
Improved user journey for Libre 2 users

### DISCUSSION THREAD

**Discussion by PhilTU** [NightscoutFoundation/xDrip#2956] [Ideas]
I would like to suggest some ways xDrip+ could be enhanced to improve the user experience of Libre 2 users, specifically those using "direct connection". As a volunteer support person I see users making errors because they are misguided by what they see, and it would be good to consolidate the many advances in xDrip+ and Libre 2 into the user experience.

1. The first change I propose is that the Connection Wizard has an option for Libre 2 Direct, with a Libre sensor icon like "Libre Patch". This makes it clear to the novice what to click, when the current option of "Libre Bridge" is represented by a cluster of hardware devices they don't have. Obviously there are still Bridge users needing that option, and Libre Patch users, so I am proposing an additional option in the wizard.

2. If the new icon is available in the Wizard then we need to refine the workflow of the wizard. Currently after selecting "Bridge" the L2 direct user is taken to an inappropriate Bluetooth scan looking for a Bridge - this should be replaced by a settings change to enable NFC scanning and the starting of L2 sensors, and a check / prompt that OOP2 is installed / working.

3. The OOP2 option "no calibration" should be selected as Default in order to remove the obfuscation of the "waiting for data" calibration screen and allow the user to establish a working connection to the sensor before considering calibration.

The goal is to get the novice user as close as possible to the correct setup, rather than have them trying to find multiple settings without on-screen guidance.

I appreciate there may be limited resources and this may not be a priority, I appreciate the work done in enabling the Libre 2 direct connection and would like to build on that to further improve the user experience.

**Comment by Navid200** [comment]
@tzachi-dar

**Comment by tzachi-dar** [comment]
I think that these are good ideas.
I don't know if I'll have time to implement them though.

**Reply by Navid200** [reply]
I can implement them.  I will ask for your review.
Thanks
